### ISSUE TITLE
Add support for Fn::ForEach

### ISSUE THREAD

**Issue by dhutchison** [DontShaveTheYak/cloud-radar#271]
New intrinsic function: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-foreach-example-resource.html

Allowed to be used in Conditions, Outputs & Resources. The documentation links to examples which helpfully includes input/output examples which we could use for testing the conversion.

**Comment by shadycuz** [comment]
I dont think this is available until you turn on Transforms? Or maybe they did away with that? 
https://aws.amazon.com/blogs/devops/exploring-fnforeach-and-fnfindinmap-enhancements-in-aws-cloudformation/

**Comment by dhutchison** [comment]
I think you are right, pretty sure I saw it in the docs. 

Not really looked into it at all, just popped in to my head as I was discussing a use case today where it might come in useful in the future.

**Comment by shadycuz** [comment]
Today I am starting work on the ability to support transforms, sponsored by my current employer 😎

**Comment by dhutchison** [comment]
Released in v0.17.0
